sql/moz-fx-data-shared-prod/telemetry_derived/newtab_interactions_v1/schema.yaml

```diff
@@ -0,0 +1,103 @@
+fields:
+- mode: NULLABLE
+  name: newtab_visit_id
+  type: STRING
+- mode: NULLABLE
+  name: client_id
+  type: STRING
+- mode: NULLABLE
+  name: submission_date
+  type: DATE
+- mode: NULLABLE
+  name: search_engine
+  type: STRING
+- mode: NULLABLE
+  name: search_access_point
+  type: STRING
+- fields:
+  - mode: NULLABLE
+    name: key
+    type: STRING
+  - fields:
+    - mode: NULLABLE
+      name: branch
+      type: STRING
+    - fields:
+      - mode: NULLABLE
+        name: type
+        type: STRING
+      mode: NULLABLE
+      name: extra
+      type: RECORD
+    mode: NULLABLE
+    name: value
```

Comment:
The schama is a bit hard to read. Could we maybe do something as follows:

- name:
  mode: 
  type:
  fields:

so sort of reading from top to bottom?